JSONEncoder.NonConformingFloatEncodingStrategy.convertToString error

Hi @mattt I'm always hitting this issue ?

How could I help fixing it ?

```
Error: invalidValue(-nan, Swift.EncodingError.Context(codingPath: [NestedCodingKeys(stringValue: "totals", intValue: nil), CodingKeys(stringValue: "percent", intValue: nil)], debugDescription: "Unable to encode Double.nan directly in JSON. Use JSONEncoder.NonConformingFloatEncodingStrategy.convertToString to specify how the value should be encoded.", underlyingError: nil))
make: *** [doc] Error 1
```

Thanks,

--- a/Sources/swift-dcov/Supporting Types/Counter.swift
+++ b/Sources/swift-dcov/Supporting Types/Counter.swift
@@ -5,7 +5,8 @@ struct Counter {
     var total: Int { hits + misses }
 
     var percentage: Double {
-        Double(hits) / Double(total) * 100.0
+        guard total > 0 else { return 0 }
+        return Double(hits) / Double(total) * 100.0
     }
 
     init(hits: Int = 0, misses: Int = 0) {

